common/compositor/compositor.cpp

```diff
@@ -33,8 +33,8 @@
 namespace hwcomposer {
 
 Compositor::Compositor() {
-  deinterlace_.flag_ = HWCDeinterlaceFlag::kDeinterlaceFlagNone;
-  deinterlace_.mode_ = HWCDeinterlaceControl::kDeinterlaceNone;
+  deinterlace_.flag_ = HWCDeinterlaceFlag::kDeinterlaceFlagAuto;
```

Comment:
I think you need to protect these assignments with lock & unlock as follows:

     lock_.lock();
     deinterlace_.flag_ = HWCDeinterlaceFlag::kDeinterlaceFlagNone;
     deinterlace_.mode_ = HWCDeinterlaceControl::kDeinterlaceNone;
     lock_.unlock();